### DISCUSSION TITLE
film negative: add features to spot picker

### DISCUSSION THREAD

**Discussion by nomar500** [Beep6581/RawTherapee#6175] [Ideas]
Hi,
 
in the context of [film negative module](https://discuss.pixls.us/t/any-interest-in-a-film-negative-feature-in-rt/12569), I would like to assess the following:
- how hard would it be to have a drop-down list to select the size of the spot? (as in the _white balance_ tool) right now, the size is hard-coded.
- how hard would it be to change this spot into an arbitrary rectangle? (this kind of selection tool exists in darktable, see *negadoctor*)
- in either case, how can a different kind of processing be composed with the _average_ set of values the spot tool already computes?
  - more specifically, due to the nature of the noise (crudely compared to salt and pepper) and the use of these spot pipettes, I would like to have something like a median filter applied to the "spot zone" before the average value is computed. 

Thanks!

cc. @rom9

**Comment by Thanatomanic** [comment]
This is not the right place for such discussion. Please file an Issue and select the 'Feature request' option.

Edit: it might be what GitHub thinks is the right place for such discussion, but not for our current workflow.

**Reply by nomar500** [reply]
savvy, done in #6178

**Comment by rom9** [comment]
Hi ! I agree with @Thanatomanic , please open a new feature request. Anyway, the first two points are already on my todo list ;-)
As for the third point, i'm not sure i understand ... i think it would be easier to just compute the median value of the area. Please can you elaborate more on this point in your feature request?
Thanks :-)

**Reply by nomar500** [reply]
done 👍 
I've left the spot size evolutions in the feature request, so it's official and not lost in `// TODO` comments 😀
